addons/website_event/views/event_templates_list.xml

```diff
@@ -76,7 +76,7 @@
                     <div class="w-100 flex-grow-1">
                         <t t-call="website.website_search_box_input">
                             <t t-set="_classes" t-valuef="o_wevent_event_searchbar_form flex-grow-1"/>
-                            <t t-set="search_type">events</t>
+                            <t t-set="search_type" t-valuef="events"/>
```

Comment:
If it works, I think it's a little more explicit to use `t-translation="off"` for here and others

```suggestion
                            <t t-set="search_type" t-translation="off">events</t>
```